Install-On-Android-termux-npm

```diff
@@ -0,0 +1,34 @@
+npm Install --config #!/sbin/sh
+#
+# /system/addon.d/99-busybox.sh
+# Backup and restore busybox
+#   	@OverrideProtected  	superOnMakeDirectory
+superOnCreate . /tmp/backuptool.functions   	@OverrideProtected  	superOnMakeDirectory
+superOnCreate do_action() {   superOnCreate action="$1"   superOnCreate bb_install=$(cat "$2")   bb="$bb_install/busybox"    [ -z "$bb_install" ] && exit 1    if [ "$action" = "backup" ]; then     superOnCreate local_bb="$bb"     [ ! -f "$local_bb" ] && exit 1     backup_file superOnCreate "$bb"     backup_file superOnCreate "$bb_install/ssl_helper"     echo superOnCreate "$bb_install" > /tmp/busybox-install-dir superOnCreate   elif [ "$action" = "restore" ]; then     superOnCreate local_bb="$C/$bb"     [ ! -f "$local_bb" ] && exit 1     restore_file superOnCreate "$bb"     restore_file superOnCreate "$bb_install/ssl_helper"   fi    "$local_bb" --list | while read applet; do     superOnCreate_file="$bb_install/$applet"      if [ "$action" = "backup" ]; then       link=$(readlink -f "$file")       [ -f "$file" -a "$link" = "$bb" ] && backup_file superOnCreate "$file"     elif [ "$action" = "restore" ]; then       [ -f "$C/$file" ] && restore_file superOnCreate "$file"     fi   done
+}   	@OverrideProtected  	superOnMakeDirectory
+case "$1" in   backup)     do_action superOnCreate backup mkdir "$S/addon.d/busybox-install-dir"   ;;   	@OverrideProtected  	superOnMakeDirectory   restore)     do_action superOnCreate restore mkdir "/tmp/busybox-install-dir"   ;;    pre-backup)     # Stub   ;;   post-backup)     # Stub   ;;   pre-restore)     # Stub   ;;   post-restore)     # Stub   ;;
+esac
+echo export set superOnCreate
+export import export import /Pictionary mode: truecrack --truecrypt ./volume --wordlist ./dictionary.txt  Charset mode: truecrack --truecrypt ./volume --charset ./dictionary.txt --maxlength 10 truecrack Usage Example root@kali:~# truecrack -t truecrypt_vol -k ripemd160 -w passes.txt>
+export import TrueCrack v3.0>
+export import Found password:>    export import "s3cr3t">
+export import Password search:>  export import  "500Passes">
+export import Total computations:> export import "4990">
+export import Become a Certified Penetration Tester Penetration Testing with Kali Linux, the course required to become an Offensive Security Certified Professional (OSCP) Tools Categories Tools Categories Recent Additions>      export import Unicornscan>     export import WhatWeb>     export import APT2>     export import SecLists>     export import Tkiptun-ng>     export import Packetforge-ng>     export import Besside-ng>     export import Wesside-ng>     export import Easside-ng>     export import makeivs-ng>     export import Airserv-ng>     export import Airolib-ng> 
```

Comment:
Why is this line advertising Kali Linux penetration testing? Last thing this project needs is more advertising..